assets/settings/default.json

```diff
@@ -688,6 +688,18 @@
     "copy_on_select": false,
     // Whether to show the terminal button in the status bar
     "button": true,
+    // Whether to show the terminal minimize button above the terminal
+    "minimize_button": true,
+    // Whether to show the terminal maximize button above the terminal
+    "maximize_button": false,
+    // Whether to show a drop down for spawning tasks & creating new terminals or creating a new terminal
+    "spawn_task_option_button": true,
+    // Whether to show the inline assistent button above the terminal
+    "inline_assist_button": true,
+    // Whether to show the buttons above the terminal only when terminal in focus
+    "require_focus_for_buttons": true,
+    // In which order the buttons above the terminal are
+    "arrange_buttons": ["InlineAssist", "New", "Minimize", "Maximize"],
```

Comment:
I don't think we should add a setting to arrange the order of the buttons.

We don't really do anything like this elsewhere, and it seems like it's going to restrict what we're able to do with the UI.

Reply:
it does, it changes the order of the button but the functionality stays the same but ok